Session/Conversations/ConversationVC.swift

```diff
@@ -465,30 +466,53 @@ final class ConversationVC : BaseVC, ConversationViewModelDelegate, OWSConversat
     // MARK: Updating
     
     func updateNavBarButtons() {
+        // Back button (to appear on pushed screen)
+        let backButton = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        backButton.tintColor = Colors.text
+        navigationItem.backBarButtonItem = backButton
+        
+        navigationItem.hidesBackButton = isShowingSearchUI
+        
         if isShowingSearchUI {
             navigationItem.leftBarButtonItem = nil
             navigationItem.rightBarButtonItems = []
-        } else {
+        }
+        else {
+            navigationItem.hidesBackButton = false
```

Comment:
This line seems to be redundant to `navigationItem.hidesBackButton = isShowingSearchUI`

Reply:
Ah good catch, will remove